src/authService.js

```diff
@@ -0,0 +1,38 @@
+// authService.js
+import axios from 'axios';
+
+export const isAuthenticated = async () => {
+  try {
+    const token = localStorage.getItem('token');
+    const formData = new FormData();
+    formData.append('token', token);
+
+    const formDataObject = {};
+    formData.forEach((value, key) => {
+      formDataObject[key] = value;
+    });
+     
+    const response = await axios.post(`http://localhost:8081/api/auth/verification`, formDataObject, {
+      headers: {
+        Authorization: `${token}`
+      }
+    });
+
+    console.log(response);
```

Comment:
remove console.

Reply:
also if we have the token then why do we need to run the verification call every time ? This verification should be done on the backend, if token is not valid then backend will reject the api call.